server/src/com/cloud/consoleproxy/ConsoleProxyManagerImpl.java

```diff
@@ -1246,8 +1246,7 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
 
         Map<String, String> configs = _configDao.getConfiguration("management-server", params);
 
-        String value = configs.get(Config.ConsoleProxyCmdPort.key());
-        value = configs.get("consoleproxy.sslEnabled");
+        String value = configs.get("consoleproxy.sslEnabled");
         if (value != null && value.equalsIgnoreCase("true")) {
```

Comment:
You can use `org.apache.commons.lang3.BooleanUtils.toBoolean(String)`.

Reply:
@rafaelweingartner I checked this is a string comparison, the 'true' has to do with the string stored in global setting's value from db. Looks like a massive cleanup may be needed in this class and others. I'll avoid changes in that case, keep the changes in PR minimal and maybe send a separate cleanup PR.